src/backend/db/categories.js

```diff
@@ -1,5 +1,8 @@
 import { v4 as uuid } from "uuid";
-
+import makeup from "../../Components/Home/Images/makeup.jpg";
+import skincare from "../../Components/Home/Images/skin.jpg";
+import haircare from "../../Components/Home/Images/hair.jpg";
+import fragrances from "../../Components/Home/Images/Fragrances.jpg";
```

Comment:
Components folder should only contain components, put images separate, as suggested above